corehq/apps/export/transforms.py

```diff
@@ -19,6 +19,37 @@ def owner_id_to_display(owner_id, doc):
 def case_id_to_case_name(case_id, doc):
     return _cached_case_id_to_case_name(case_id)
 
+
+def workflow_transform(workflow, doc):
+    from corehq.apps.sms.models import (
```

Comment:
we'll need to include WORKFLOW_FORWARD in the relevant_workflows below, looks like it's actually also missing in the existing export process